packages/klighd-core/src/views.tsx

```diff
@@ -39,10 +40,16 @@ export class SKGraphView implements IView {
 
     @inject(KlighdInteractiveMouseListener) mListener: KlighdInteractiveMouseListener
     @inject(DISymbol.RenderOptionsRegistry) renderOptionsRegistry: RenderOptionsRegistry
+    @inject(TYPES.IActionDispatcher) private actionDispatcher: IActionDispatcher;
 
     render(model: Readonly<SGraph>, context: RenderingContext): VNode {
         const ctx = context as SKGraphModelRenderer
-        ctx.renderingDefs = new Map
+        this.actionDispatcher.dispatch(SendModelContextAction.create(model, ctx))
+
+        if (!ctx.renderingDefs) {
+            // Make sure not to create a new map all the time
```

Comment:
Could this be adding the font defs, even if they already exist though?

Reply:
Since it's a map they would just be replaced as it was done previously

Reply:
Only if the IDs are the same, I would take care with this if this can cause a new issue.